sdk/communication/AzureCommunicationCommon/CHANGELOG.md

```diff
@@ -64,5 +70,5 @@ our efforts can be found in the
 
 ### Added Libraries
 
-- Azure Communication Services common ([AzureCommunication](https://github.com/Azure/azure-sdk-for-ios/tree/master/sdk/communication/AzureCommunication))
+- Azure Communication Services common ([AzureCommunication](https://github.com/Azure/azure-sdk-for-ios/tree/master/sdk/communication/AzureCommunicationCommon))
```

Comment:
```suggestion
- Azure Communication Services common ([AzureCommunication](https://github.com/Azure/azure-sdk-for-ios/tree/master/sdk/communication/AzureCommunication))
```
I think this can be undone cause it is older in the changelog

Reply:
The problem is that we have a link-checker and if we don't update the link, it will be broken.